nntoolkit/create.py

```diff
@@ -35,6 +35,62 @@ def get_parser():
                         metavar="MODEL_FILE")
     return parser
 
+def xaviar10_weight_init(neurons_a, neurons_b):
+    fan_in = neurons_a
+    fan_out = neurons_b - 2
+    init_weight = 4.0*numpy.sqrt(6.0/(fan_in+fan_out))
+    _W = [numpy.random.uniform(low=-init_weight, high=init_weight, size=neurons_a) \
```

Comment:
Why did you use an underscore? I usually use underscore prefixes for private attributes of objects.